src/lib/client.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiService } from './api/api.service';
+import { AuthResource } from './resources/auth.resource';
+
+export interface PermanentConstructorConfigI {
+  sessionToken: string;
+  mfaToken: string;
+  accountId: number;
+  archiveId: number;
```

Comment:
Are either `accountId` or `archiveId` used? They don't appear to be, but then I'm confused about how users are able to log in; is all the login data in `sessionToken`? Do we need both `sessionToken` and `accountId`?

Reply:
Yep, you're right, I was just starting to realize this! `sessionToken` covers all that server-side, and the `accountId` would only be needed in some use cases that this SDK definitely won't be covering at first (and most likely never will!)